infra/scripts/test-end-to-end-batch.sh

```diff
@@ -113,9 +113,21 @@ feast:
   tracing:
     enabled: false
 
+  security:
```

Comment:
Is it really necessary to add these values? Can defaults not be used?

Reply:
The `classpath:/application.yml` was not being passed on the feast-serving start up, when I made this change. I see that it's been added. defaults can be used now.